### PR TITLE
Call to action pattern

### PR DESCRIPTION
**Pull request by davidcmoulton**
Still needs JavaScript.

### CODE REVIEW

File: source/_patterns/01-molecules/components/call-to-action.mustache [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<div class="call-to-action-wrapper" id="{{id}}">
+
+  <div class="call-to-action">
+
+    {{#image}}
```
Comment (thewilkybarkid) [inline source/_patterns/01-molecules/components/call-to-action.mustache]: Shouldn't be needed?

File: assets/sass/patterns/molecules/call-to-action.scss [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    }
+
+    grid-column: 3;
+    grid-row: 1;
+  }
+
+}
+
+.call-to-action__close {
+  // TODO: Update when js injects .call-to-action__close into the DOM instead of it being created by the template
```
Comment (thewilkybarkid) [inline assets/sass/patterns/molecules/call-to-action.scss]: Can we just remove for now?

Comment (davidcmoulton) [reply assets/sass/patterns/molecules/call-to-action.scss]: Gone.

File: source/_patterns/01-molecules/components/call-to-action.mustache (hunk 1 above)
Comment (davidcmoulton) [reply source/_patterns/01-molecules/components/call-to-action.mustache]: It's gone.

**Pull request merged**